android/KMEA/app/src/main/java/com/tavultesoft/kmea/cloud/impl/CloudCatalogDownloadCallback.java

```diff
@@ -218,7 +218,7 @@ public void processCloudReturns(Dataset aDataSet, CloudCatalogDownloadReturns js
       } // else no match == no special handling.
     }
 
-    if (updateBundles.size() > 0) {
+    if (updateBundles.size() > 0 && !(DEBUG_SIMULATE_UPDATES && !executeCallbacks)) {
```

Comment:
Can you add a note in line 33 that  `DEBUG_SIMULATE_UPDATES` is used for testing (like `FORCE_RESOURCE_UPDATE`)